src/test/java/redis/clients/jedis/tests/JedisSentinelPoolWithCompleteCredentialsTest.java

```diff
@@ -178,12 +176,9 @@ public void customClientName() {
     JedisSentinelPool pool = new JedisSentinelPool(MASTER_NAME, sentinels, config, 1000,
         "default", "foobared", 0, "my_shiny_client_name");
 
-    Jedis jedis = pool.getResource();
-
-    try {
+    try (Jedis jedis = pool.getResource()){
       assertEquals("my_shiny_client_name", jedis.clientGetname());
     } finally {
-      jedis.close();
       pool.destroy();
```

Comment:
another try with resources
```suggestion
      pool.close();
```